packages/integrations/image/src/vendor/squoosh/emscripten-utils.ts

```diff
@@ -10,19 +10,14 @@ export function pathify(path: string): string {
 
 export function instantiateEmscriptenWasm<T extends EmscriptenWasm.Module>(
   factory: EmscriptenWasm.ModuleFactory<T>,
-  path: string,
-  workerJS = ''
+  bytes: Uint8Array,
 ): Promise<T> {
   return factory({
-    locateFile(requestPath) {
-      // The glue code generated by emscripten uses the original
-      // file names of the worker file and the wasm binary.
-      // These will have changed in the bundling process and
-      // we need to inject them here.
-      if (requestPath.endsWith('.wasm')) return pathify(path)
-      if (requestPath.endsWith('.worker.js')) return pathify(workerJS)
-      return requestPath
-    },
+		// @ts-expect-error
+		wasmBinary: bytes,
```

Comment:
The emscripten wrapper code accepts this property and it's the way to provide raw bytes, but the TS types do not allow it.

Reply:
I think it would be nice to have this comment in the code comment!